core/object/script_language_extension.h

```diff
@@ -630,7 +630,11 @@ VARIANT_ENUM_CAST(ScriptLanguageExtension::CodeCompletionLocation)
 
 class ScriptInstanceExtension : public ScriptInstance {
 public:
-	const GDExtensionScriptInstanceInfo *native_info;
+	const GDExtensionScriptInstanceInfo2 *native_info;
+	struct {
+		GDExtensionClassNotification notification_func;
```

Comment:
Why not GDExtensionScriptInstanceNotification?

Reply:
Ack, yes, you're right, this should be `GDExtensionScriptInstanceNotification`. Luckly, the C types of the arguments are equivalent, so I don't think this breaks anything, but it should be fixed.

Reply:
Is this just a straightforward rename from `GDExtensionScriptInstanceInfo2` to `GDExtensionScriptInstanceNotification`? In that case I can create a PR. Not sure about other implications.

Reply:
Yep, it should just be changing the type